clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/cache/Keys.java

```diff
@@ -91,8 +95,12 @@ public static String getClusterKey(String account, String region, String cluster
     return ID + SEPARATOR + Namespace.ECS_CLUSTERS + SEPARATOR + account + SEPARATOR + region + SEPARATOR + clusterName;
   }
 
-  public static String getTaskKey(String account, String region, String taskName) {
-    return ID + SEPARATOR + Namespace.SERVICES + SEPARATOR + account + SEPARATOR + region + SEPARATOR + taskName;
+  public static String getTaskKey(String account, String region, String taskId) {
```

Comment:
This is not a getter method, please don't use "get" in the method name when the method is not just getting

Reply:
Following what everyone is doing for naming these methods.